docs/csharp/tutorials/inheritance.md

```diff
@@ -275,7 +275,7 @@ We can now instantiate a `Book` object, invoke both its unique and inherited mem
 
 In the previous example, we defined a base class that provided an implementation for a number of methods to allow derived classes to share code. In many cases, however, the base class is not expected to provide an implementation. Instead, the base class is an *abstract class*; it serves as a template that defines the members that each derived class must implement. Typically in the case of an abstract base class, the implementation of each derived type is unique to that type.
```

Comment:
What's confusing here is that, since we applied the `abstract` keyword to the Publication class in the last section, it would seem to be an abstract class. I'd add a one sentence qualification after the first sentence, something like, "We marked the class with the `abstract` keyword because it made no sense to instantiate a Publication object, although the class did provide implementations of functionality common to publications."